src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -722,10 +728,11 @@ private static ClassLoader contextClassLoader(Thread parent) {
         }
 
         if (attached && VM.initLevel() < 1) {
-            this.tid = 1;  // primordial thread
+            this.tid = 3;  // primordial thread
```

Comment:
The comment before the `ThreadIdentifiers` class needs updating to account for this change.

Reply:
Fixed.